data-raw/process_data_sets.R

```diff
@@ -272,3 +272,12 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+
+## Adding international powerlifting data
+ipf_lifts <- 
+  "data-raw/IPF Lifts - ipf_lifts.csv" %>%
+  read_csv() %>%
+  clean_names()
```

Comment:
Look at the data types of the variables. `sex`, `equipment`, `event` all should be factors, not characters. This is not an exhaustive list of changes.